lib/model/knexfile.js

```diff
@@ -0,0 +1,33 @@
+// Copyright 2021 ODK Central Developers
+// See the NOTICE file at the top-level directory of this distribution and at
+// https://github.com/getodk/central-backend/blob/master/NOTICE.
+// This file is part of ODK Central. It is subject to the license terms in
+// the LICENSE file found in the top-level directory of this distribution and at
+// https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+// including this file, may be copied, modified, propagated, or distributed
+// except according to the terms contained in the LICENSE file.
+
+/*
+This file is for use with the Knex migration CLI. Run `npx knex` from the
+repository root, specifying the path of this file to --knexfile. Because the
+Knex CLI will change the working directory to the directory of this file
+(lib/model), you will also need to specify the NODE_CONFIG_DIR environment
```

Comment:
This environment variable is documented here: https://github.com/lorenwest/node-config/wiki/Environment-Variables#node_config_dir

Reply:
As an alternative to setting `$NODE_CONFIG_DIR`, I also tried using the `--cwd` option of the Knex CLI. However, that didn't work: the working directory was still changed to `lib/model` before `config` was `require`d.